docs/getstarted.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible"
+        content="IE=edge" />
+    <meta name="viewport"
+        content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/f6863db2d0.js"
+        crossorigin="anonymous"></script>
+    <link rel="stylesheet"
+        href="../styles.css" />
+    <title>ClutchUI | Get Started</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <!-- navigation -->
+        <div class="nav">

```

Comment:
you can use "header" tag here

Reply:
Thanks for pointing it out. Will change it.